interface.py

```diff
@@ -0,0 +1,5 @@
+from collections import namedtuple
```

Comment:
While named tuples are nice, we'll usually work with dicts, because they can be easily translated to JSON later on.

Reply:
Not my code - from the exercise documentation